ui/src/preview/translations.fi.json

```diff
@@ -79,6 +79,9 @@
                 "state": {
                     "PLANNED": "Ratanumero on tilassa \"Suunniteltu\""
                 },
+                "reference-line": {
+                    "null": "Ratanumerolta puuttuu pituusmittauslinja tai se ei ole mukana julkaisussa"
```

Comment:
Yhtenäistin nämä viestit nyt kaikki tähän muottiin joka vihjaa jo siihen todennäköisimpään syyhyn eli että linja ei ole mukana julkaisusetissä

Reply:
Testatessa tuli mieleen, että pitäisikö ratanumeroilla ja pituusmittauslinjoilla tältä virheeltä jättää pois tuo "puuttuu pituusmittauslinja/ratanumero" -osuus ja vain sanoa tyyliin "Ratanumeron pituusmittauslinja ei ole mukana julkaisussa"? Meillähän ei oikeastaan pitäisi olla tilanteita, joissa nuo menevät sillä tavalla epäsynkkaan että toinen löytyy ja toista ei (vaikka kantarajoitteet sen taisivat ehkä mahdollistaa, että pituusmittauslinja löytyisi ilman ratanumeroa).

Reply:
Joo tai siis toisin päin: kannassa voi olla ratanumero ilman pituusmittauslinjaa mutta toisin päin se ei ole mahdollista. Joka tapauksessa, ihan validi pointti ja mietin itse samaa kirjiottaessa, mutta en ollut varma ja jätin geneeriseksi. Voisi  tuon kyllä muuttaa, ei sen pitäisi väärin mitään sanoa vaikka rakenne vähän muuttuisikin.